packages/core-components/src/components/Select/Select.tsx

```diff
@@ -136,6 +136,7 @@ export type SelectProps = {
   native?: boolean;
   disabled?: boolean;
   margin?: 'dense' | 'none';
+  testid?: string;
```

Comment:
Just a minor renaming comment 🙂 :
I would recommending renaming the property to `data-testid` as it is a convention used by components libraries such as Material UI and React testing library, so it feels a intuitive pattern.